### PR TITLE
Remove additional filters/ordering when fetching messages by id

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

**Comment by nicpottier** [comment]
Nice, does make me wonder if we should be approaching this in the other direction though.. (remove order by attribute on model and making places that care about ordering specify it)

**Comment by nicpottier** [comment]
API calls ok now as well?

**Comment by rowanseymour** [comment]
@nicpottier they have explicit ordering - am currently firing up a test database to check the inbox views

File: temba/flows/models.py [hunk 1]
```diff
@@ -4568,10 +4568,13 @@ def get_contact_urn_display(contact, scheme=None):
         for run_step in ChunkIterator(FlowStep, step_ids,
                                       order_by=['contact', 'run', 'arrived_on', 'pk'],
                                       select_related=['run', 'contact'],
-                                      prefetch_related=['messages__contact_urn',
-                                                        'messages__channel',
-                                                        'broadcasts',
-                                                        'contact__all_groups'],
+                                      prefetch_related=[
+                                          Prefetch('messages', queryset=Msg.objects.order_by('-id')),
```
Comment (rowanseymour) [inline temba/flows/models.py]: Need to give this explicit order because later we call 'first()' on it to get the last message

**Pull request merged**